src/yoroi-wallets/utils/parsing.ts

```diff
@@ -70,6 +70,11 @@ export const parseBoolean = (data: unknown) => {
   return isBoolean(parsed) ? parsed : undefined
 }
 
+export const asciiToHex = (text: string) => {
```

Comment:
move to nfts.ts

Reply:
This is a general `asciiToHex` helper, is `nfts.ts` the best place for this util?